### DISCUSSION TITLE
Сделай с помощью OpenCV проход капчи!

### DISCUSSION THREAD

**Discussion by Northio-Sodium** [Titlehhhh/Minecraft-DDOS-Free#35] [Ideas]
Будет круто DDoS-ить сервера с капчей. Для этого я бы посоветовал OpenCV на Python. Скорость у него высокая.
С помощью Keyboard можно управлять клавиатурой.
А YOLO может прочитать текст с экрана. Но только нужно куча картинок капчи чтобы собрать дата-сет.
Успехов!

**Comment by Titlehhhh** [comment]
Против каптч ничего не имею.
Мне кажется для каптч yolo и opencv слишком "тяжёлые". А если и не тяжёлые, то нужно c++ использовать или c#, так как с Python нужно будет общаться по IPC или сокетам, а это лишний оверхед.
